versions/baseline.json

```
@@ -1362,7 +1362,7 @@
     },
     "cairo": {
       "baseline": "1.17.8",
-      "port-version": 2
+      "port-version": 4
```

```suggestion
      "port-version": 3
```
You need to run the command `./vcpkg x-add-version cairo`